index.html

```diff
@@ -2445,24 +2447,92 @@ <h2>Verification Algorithm</h2>
 As a general rule, verifiers SHOULD strive to minimize the processing of untrusted data.
 This includes minimizing any processing of the protected header, unprotected header, or payload as part of the key discovery procedures.
     </p>
-
     <p>
-When verifying a credential or presentation secured with SD-JWT, the algorithm defined in
-[[SD-JWT]] for
-  Verification of the SD-JWT
-MUST be followed.
+After verification has succeeded, additional validation checks SHOULD be
+performed as described in Section <a href="#validation-algorithm"></a>
     </p>
-
+    <p>
+The outputs for the following algorithms are:
+    </p>
+    <ul>
+      <li>
+<code>status</code>: a boolean indicating the result of verification,
+<code>true</code> for success and <code>false</code> for failure.
+        </li>
+      <li>
+<code>document</code>: a document conforming to the [[VC-DATA-MODEL-2.0]]
+        </li>
+      <li>
+<code>mediaType</code>: <code>vc+ld+json</code>
+        </li>
+    </ul>
+    <section>
+      <h3>Algorithm for verifying a credential or presentation secured with SD-JWT</h3>
+      <p>
+The inputs for this algorithm are:
+      </p>
+      <ul>
+        <li>
+<code>inputMediaType</code>: <code>vc+ld+json+sd-jwt</code>
+          </li>
+        <li>
+<code>inputDocument</code>: the verifiable credential secured with [[SD-JWT]]
+          </li>
+      </ul>
+      <p>
+Upon receipt of the verifiable credential secured with [[SD-JWT]], the holder or
+verifier follows this algorithm:
+      </p>
+        <ol>
+          <li>
+Follow the algorithm defined in [[[SD-JWT]]] for Verification of the SD-JWT.
+          </li>
+          <li>
+If processing aborts for any reason or the SD-JWT is rejected:
+            <ol>
+              <li>
+Set <code>status</code> to <code>false</code>
+              </li>
+              <li>
+Set <code>document</code> to <code>null</code>
+              </li>
+              <li>
+Set <code>mediaType</code> to <code>null</code>
+              </li>
+              <li>
+Return
+              </li>
+            </ol>
+          </li>
+          <li>
+If processing completes successfully:
+            <ol>
+              <li>
+Set <code>status</code> to <code>true</code>
+              </li>
+              <li>
+Set <code>mediaType</code> to <code>vc+ld+json</code>
+              </li>
+              <li>
+Convert the SD-JWT payload back into the JSON claim set by reversing the process
+described in [[[SD-JWT]]]. Set <code>document</code> to the JSON claim set.
```

Comment:
```suggestion
described in [[[SD-JWT]]] [[SD-JWT]]. Set <code>document</code> to the JSON claim set.
```

Reply:
Again, why do we want both triple brackets and double brackets?